fab/src/main/res/drawable/fab_cover_circle.xml

```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape
-	xmlns:android="http://schemas.android.com/apk/res/android"
-	android:shape="oval">
-
-	<solid android:color="#99ffffff"/>
-
-</shape>
\ No newline at end of file
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape android:shape="oval">
+            <solid android:color="#eee" />
+        </shape>
+    </item>
+    <item>
+        <shape android:shape="oval">
+            <solid android:color="#ffffffff" />
+        </shape>
+    </item>
+</selector>
```

Comment:
Please use 6- or 8-digit colour codes to keep consistency with the rest of the project.